packages/system/src/reset.ts

```diff
@@ -18,7 +18,7 @@ const block = css({
 });
 
 const list = css({
-  listStyle: 'none',
+  // empty
```

Comment:
would you not rather leave it without comment?

Reply:
🤷‍♂️